Components/Image/image.css

```diff
@@ -0,0 +1,30 @@
+:root {
+    --black: #000000;
+    --medium-font-size: medium;
+    --large-font-size: large;
+    --larger-font-size: larger;
+    --bolder-font-weight: bolder;
```

Comment:
i suggest instead of using medium, larger you can use value.

Reply:
Yeah, I have updated them 👍🏻